libsolidity/interface/StandardCompiler.cpp

```diff
@@ -91,6 +92,21 @@ Json::Value formatErrorWithException(
 	return formatError(_warning, _type, _component, message, formattedMessage, location);
 }
 
+/// @_hash contains a Keccak256 hash with the 0x prefix
```

Comment:
Actually, we can make a decision whether the hash should be prefixed with `0x` or not. Usually hashes are not prefixed.